src/services/motor.ts

```diff
@@ -35,13 +40,20 @@ export class Range extends HyundaiService {
       this.maxRange = this.currentRange;
       this.log.info(`maxRange is ${this.maxRange}`);
     }
+    if (this.batteryChargeHV !== status.engine.batteryChargeHV) {
+      this.batteryChargeHV = status.engine.batteryChargeHV;
```

Comment:
Looks like you found something in bluelinky that's better than range, I'm having trouble following it though. Can you clarify for me where this is better than range?

https://github.com/Hacksore/bluelinky/blob/9dedd29f7ec61efc3ce8e9da99632c001e8d1963/src/vehicles/american.vehicle.ts#L212

Reply:
Makes sense, the field you added is more accurate for EV battery %, and the range logic is a fallback for non EV